go.mod

```diff
@@ -133,3 +133,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/go-jose/go-jose/v4 => github.com/go-jose/go-jose/v4 v4.0.1
```

Comment:
Why?

Reply:
Are the last two commits (updating cert-manager and this change) an attempt to fix the vulnerability detected by snyk?

Reply:
Yes

Even this was an attempt to fix it. I will be removing it before merging